docs/source/development/traffic_portal.rst

```diff
@@ -44,6 +44,40 @@ Installing The Traffic Portal Developer Environment
 #. Navigate to the ``traffic_portal`` subdirectory of your cloned repository.
 #. Run ``npm install`` to install application dependencies into ``traffic_portal/node_modules``. Only needs to be done the first time unless ``traffic_portal/package.json`` changes.
 #. Run ``bower install`` to install client-side dependencies into ``traffic_portal/app/bower_components``. Only needs to be done the first time unless ``traffic_portal/bower.json`` changes.
+#. Make sure that compass is installed and functioning correctly by running ``compass version``. If ``compass`` is not running, you can install it on macOS as follows:
+    #. macOS comes with its own version of ruby built into it. In order to install compass, if you run a command like ``sudo gem install compass``, it will take the default ruby available in mac. In order to properly install compass, here is what you should be doing:
+
+    #. ``brew install ruby``
+
+    #. ``gem update --system``
+
+    #. ``gem install compass``
+
+        .. tip:: You need to install ``sass`` before you install ``compass``. You can do that by running:
+                 ``gem install sass``
+
+    #. Make sure that ``compass`` and ``sass`` are part of your ``PATH`` environment variable.
+
+    #. If not, you can see where gem installs ``compass`` and ``sass`` by running:
+       ``gem environment``
+
+    #. In there, you can see where ruby is installing all the gems. Add that path to your ``PATH`` environment variable.
+       For example, it is ``/usr/local/lib/ruby/gems/2.7.0/gems/compass-1.0.3/bin/`` for this test setup.
+
+    #. Once you have installed ``compass`` successfully, make sure you can reach it by typing:
+       ``compass version``
+       This should give a valid output. For example, for the test setup, the output is:
+
+    .. code-block:: text
+
+        :caption: Compass version output
```

Comment:
Directives syntactically work like this:
```rst
.. directive:: arguments
    :option: option arguments if it has any

    content
```
note the blank line goes between the directive/options and the actual _content_, and there isn't one between the directive and the _options_. That's the only way reST can tell whether something is an option or part of the content.

So tl;dr you gotta get rid of that blank line.